modules/logbucket/main.tf

```diff
@@ -33,10 +33,23 @@ resource "google_project_service" "enable_destination_api" {
 #------------#
 
 resource "google_logging_project_bucket_config" "bucket" {
-  project        = google_project_service.enable_destination_api.project
-  location       = var.location
-  retention_days = var.retention_days
-  bucket_id      = var.name
+  project          = google_project_service.enable_destination_api.project
+  location         = var.location
+  retention_days   = var.retention_days
+  enable_analytics = var.enable_analytics
+  bucket_id        = var.name
+}
+
+#-------------------------#
+# Linked BigQuery dataset #
+#-------------------------#
+
+resource "google_logging_linked_dataset" "linked_dataset" {
```

Comment:
@daniel-cit can u plz also add `description` and `parent` in this resource?

Reply:
done